### PR TITLE
Feature: Card component

### LINKED ISSUE #24: Feature: Card component
# Completed card component

Feature: Card component

## Epic Description

This Feature will add acard components with all the variants

## List of Tasks (Complete in order)

- [x] #41
- [x] Cards with dismiss
- [x] Cards with text overlay
- [x] Text only cards
- [x] Vertical/Horizontal cards
- [x] Cards with shadow

### PR DESCRIPTION
**Pull request by Pratham82**
# Completed card component

## Changes made

- [x] Updated some utility classes
- [x] Added new root classes
- [x] Added Card structure
- [x] Added card styles

closes #24

### CODE REVIEW

**Comment by Manishsharma203** [comment]
Looking good to go. 
One suggestion could be to make pointer none for card with overlay if that's ok.

**Comment by Pratham82** [comment]
> Looking good to go. One suggestion could be to make pointer none for card with overlay if that's ok.

Thanks, I've added styles for disabled button.

**Comment by Manishsharma203** [comment]
> > Looking good to go. One suggestion could be to make pointer none for card with overlay if that's ok.
> 
> Thanks, I've added styles for disabled button.

Looking Great now

File: src/components/card/card.css [hunk 1]
```diff
@@ -1 +1,135 @@
...
+.card-pricing {
+  margin: var(--u-6);
+}
+
+.card-overlay-text-div {
+  position: absolute;
+  z-index: 2;
+  margin-top: var(--u-6);
+  padding: 7.75em 0;
+  background-color: #d0d9e494;
```
Comment (krgarima) [inline src/components/card/card.css]: You forgot to add CSS color variable here

**Review by krgarima — COMMENTED** [review]
prath
Overall, the site looks good.
Just check for browser compatibility (screenshot taken from chrome browser).

File: src/components/card/card.css (hunk 1 above)
Comment (Pratham82) [reply src/components/card/card.css]: Added it

**Comment by Pratham82** [comment]
> prath Overall, the site looks good. Just check for browser compatibility (screenshot taken from chrome browser).

I'm not able to recreate this issue, tried firefox and chrome, with different screen sizes. Maybe issue with preview, CSS not able to load properly?

**Comment by Pratham82** [comment]
Merging for now, I'll check the if the issue persists @krgarima  Thank you for the review

**Pull request merged**